packages/blade/docs/guides/Usage.stories.mdx

```diff
@@ -1,72 +1,99 @@
+
+<br />
+
+## Wrap your App with `BladeProvider`
+
+The first step to get all the capabilities of Blade is to wrap your Application root under Blade's Provider.
+
+Blade offers 2 pre-built themes out of the box:
+
+1. `paymentTheme` - To be used in Payment(PG) products.
+2. `bankingTheme` - To be used in RazorpayX products.
+
+The benefits of using `BladeProvider`?
+
+- Your components will automatically become responsive on different breakpoints.
+- Capability to switch between light mode and dark mode.
+- Units for typography and other layouts(where to use `px`, `rem` etc. will automatically be handled).
+
+```jsx
+// App entry point
+import { BladeProvider } from '@razorpay/blade/components';
+import { paymentTheme } from '@razorpay/blade/tokens';
+import App from from './App';
+
+function AppWrapper(): JSX.Element {
+  return (
+    <React.Fragment>
+      <BladeProvider themeTokens={paymentTheme} colorScheme="light">
+        <App />
+      </BladeProvider>
+    </React.Fragment>
+  );
+}
+
+export default AppWrapper;
+```
+
+## Mapping Components from Figma to Blade in your code
+
+Blade is built with **"What you see in Figma is what you get on Code" ** philosophy.
+
+Whenever you get a UI mockup handed over by your designer select the component you want to build. for eg: let's say I want to implement the Export button component from the below mockup.
+
+<img src={figmaComponent} alt="Figma mockup showing how to pick component and it's properties" />
+
+So to know the blade component's name and properties
+
+1. Select the component on Figma
+2. Click on the `locator` icon that will take you to component's properties
+3. Look at the component properties and copy it in code because the same properties exists on the components shipped.
+
+```jsx
+// in your file where you want to implement this
+import { Button, DownloadIcon } from '@razorpay/blade/components';
+
+<Button type="secondary" size="medium" icon={DownloadIcon} iconPosition="left">
+  Export
+</Button>;
+```
+
+## Mapping Tokens from Figma to Blade in your code
+
+> Tokens are an integral part of design system and are used to store design decisions behind a token(variable) name. That also make things easier to update without you making any change. For this reason **never hardcode** token values in your code but rather use the token name.
+
+As mentioned earlier everything that you see on Figma built using Blade you'll always find that in code as well.
+
+Let's see how you can spot a token name from figma and write it in your code
+
+<img src={figmaToken} alt="Figma mockup showing how to pick token" />
+
+Whenever you get a UI mockup handed over by your designer
+
+1. Select the component whose color token you want to use in code up
+2. Click on the inspect panel on the left and click on the token name `Surface/Background/Level3/lowContrast`.
+   > Remember, don't select the hardcoded `hsla` color value.
```

Comment:
We can make it more noticeable

```suggestion
   > **Note**
   >
   > Remember, don't select the hardcoded `hsla` color value.
```

Reply:
this won't work because this is mdx of storybook and not GitHub's md 😞

Reply:
Oh, I thought it also parsed markdown so atleast it would bold the note